src/runtime_src/core/tools/xbreplay/src/main.cpp

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2024 Advanced Micro Devices, Inc. All rights reserved.
+
+#include "seq_reconstructor.hpp"
+#include "utils/cmd_args.hpp"
+#include "utils/logger.hpp"
+
+#include <iostream>
+#include <memory>
+namespace {
+using namespace xrt::tools::xbreplay;
+
+class successfull_termination : public std::exception
+{
+  public:
+  const char* what() const noexcept override
+  {
+    return "Program terminated successfully";
+  }
+};
```

Comment:
This is abuse of exceptions :-)   Never use exceptions for success.

Reply:
removed